dashboard/app/models/user.rb

```diff
@@ -1487,18 +1487,21 @@ def verified_instructor?
   end
 
   AI_TUTOR_EXPERIMENT_NAME = 'ai-tutor'
+  AI_TUTOR_DISABLED_BY_DCDO = DCDO.get('ai-tutor-disabled', false)
```

Comment:
will there be a delay in updating this value since it is saved as a constant outside of the function? Should we just call dcdo directly in the functions below?